### PR TITLE
Update ua.m3u

### PR DESCRIPTION
**Pull request by freearhey**

### CODE REVIEW

**Review by Dum4G — DISMISSED** [review]

File: streams/ua.m3u [hunk 1]
```diff
@@ -156,3 +156,45 @@ https://tuso.cdn-01.cosmonova.net.ua/hls/tuso_ua.m3u8
...
+#EXTINF:-1 tvg-id="",Белсат HD
+http://portal.ott.pink/watch/149/video.m3u8?geo=auto&token=CFEADF9789D77A45B3B359EE168CCCA6
+#EXTINF:-1 tvg-id="",Аверс
+http://portal.ott.pink/watch/356/video.m3u8?geo=auto&token=CFEADF9789D77A45B3B359EE168CCCA6
+#EXTINF:-1 tvg-id="",Армія ТВ HD UA
+http://portal.ott.pink/watch/7572/index.m3u8?geo=auto&token=CFEADF9789D77A45B3B359EE168CCCA6
+#EXTINF:-1 tvg-id="",Галичина HD
+http://portal.ott.pink/watch/116/index.m3u8?geo=auto&token=CFEADF9789D77A45B3B359EE168CCCA6
+#EXTINF:-1 tvg-id="",Полтавское ТВ
+http://cdn10.live-tv.od.ua:8081/hrpl/720p/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/ua.m3u:194]: http://cdn10.live-tv.od.ua:8081/hrpl/hrpl-abr/playlist.m3u8

File: streams/ua.m3u [hunk 2]
```diff
@@ -156,3 +156,45 @@ https://tuso.cdn-01.cosmonova.net.ua/hls/tuso_ua.m3u8
...
+#EXTINF:-1 tvg-id="",СК1 Житомир
+http://cdn10.live-tv.od.ua:8081/sk1zt/sk1zt720p/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/ua.m3u:196]: http://cdn10.live-tv.od.ua:8081/sk1zt/sk1zt-abr/playlist.m3u8

File: streams/ua.m3u [hunk 3]
```diff
@@ -156,3 +156,45 @@ https://tuso.cdn-01.cosmonova.net.ua/hls/tuso_ua.m3u8
...
+#EXTINF:-1 tvg-id="",Сфера ТВ
+http://cdn10.live-tv.od.ua:8081/sferarv/720p/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/ua.m3u:198]: http://cdn10.live-tv.od.ua:8081/sferarv/sferarv-abr/playlist.m3u8

**Review by BellezaEmporium — COMMENTED** [review]
See comments.

+ idk about the token durability but i'm not sure it'll last.

**Comment by freearhey** [comment]
Updated.

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**